src/util.ts

```diff
@@ -108,3 +110,26 @@ export async function readJsonIfExists<T>(filepath: string):
   }
   return undefined;
 }
+
+/**
+ * Delete all files matching any of the given glob patterns, rooted in the given
+ * directory, excluding any file in node_modules/ or bower_components/.
+ */
+export async function deleteGlobsSafe(
+    globs: Iterable<string>, cwd: string): Promise<void> {
```

Comment:
I try to be paranoid with accepting `Iterable` and explicitly rule out strings, especially with `Iterable<string>` since a string is one and even the type system won't help you out if there's a mixup.

Reply:
Modulizer uses `Iterable<string>` a lot in its configuration interfaces, so to do this consistently would mean changing a bunch of other places too: https://github.com/Polymer/polymer-modulizer/blob/master/src/conversion-settings.ts#L106